src/components/SignIn/index.jsx

```diff
@@ -0,0 +1,191 @@
+
+import * as firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+import db from '../../firebaseConfig';
+import React, { Component } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+
+class SignIn extends Component {
+ 
+  handleSubmit = (e) => {
+    debugger
```

Comment:
I don't know why you are using debugger, can you explain it to me?

Reply:
> I don't know why you are using debugger, can you explain it to me?

Oh, there was something wrong about sending request. I used it to find the bug. I will delete it